docs/contributor_guide/seed_scenarios.rst

```diff
@@ -0,0 +1,85 @@
+You're a member of a team trying to measure brand sentiment from online comments and reviews.  The team hopes to use an existing language model, a third-party service for flagging abusive comments, and then train a more targeted sentiment classifier for your brand on top.
+`Hutchinson et al. (2020) <https://arxiv.org/pdf/2005.00813.pdf>`_
+
+Employment evaluations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A potential client asks you if ML can help with predicting job candidates' suitability for jobs based on a combination of personality tests and body language
+`Rhagavan et al. (2019) <https://arxiv.org/pdf/1906.09208.pdf>`_
+
+Rankings for image search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're on a team working on improving an image search system after receiving some complaints from users related to fairness.  Users often use this system to find a selection of stock images to use when making multimedia presentations.  In this system, requests start with some information about the context and user creating the query, and your team is trying to incorporate ideas about fairness like diversity and inclusision into how search results are ranked.
+`Mitchell et al. (2020) <https://arxiv.org/pdf/2002.03256.pdf>`_
+
+Sales leads for car loans
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You work at CarCorp, a company that collects special financing data: information on people who need car financing but have either low credit scores or limited credit histories, and sells this data to auto dealer as sales leads.  CarCorp  serves dealers across the United States.  A new project manager asks about leveraging data science to “improve the quality” of leads so that dealers to not churn.  CarCorp has a large amount of historical lead data (2 million leads in 2017 alone), but relatively less data on which leads had been approved for special financing (let alone why the loan was approved).
+`Passi and Barocas (2019) shttps://arxiv.org/ftp/arxiv/papers/1901/1901.02547.pdf>`_
+
+Predictive policing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You are a contractor working with the police department in a large city.  One of the project leaders in the department would like to construct a risk score for people who are known gang members engaging in knife crime.  It's important to them that they can understand what the model is doing, and are way that any model will pick up on protected characteristics.
+`Veale et al. (2018) <https://arxiv.org/pdf/1802.01029.pdf>`_
+
+Scheduling maintenance within a factory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You work within a manufacturing company, and are starting a new project that will create a schedule assigning employees to check and update certain components of the machinery to prevent critical operation failures. The component assignment is based on data that show how often different components have worn out and broken down in the past.
```

Comment:
I'm a little hazy on the harms here as well. Obviously employee scheduling has the potential for harm, but it appears that the model is more about predicting failures in the machinery?

Reply:
I asked the same question: https://github.com/fairlearn/fairlearn/pull/491#discussion_r446710871
I think it's about creating "worse" schedules for people from one group vs. another, although I don't quite know how you'd rate the quality of a schedule.

Reply:
@riedgar-ms @romanlutz Yeah, these are great questions!  There are a lot of seed scenarios here, so hopefully tomorrow the group can walk through them for part of the meeting, and then vote on the top three, and discuss **Real harms** and great questions like this, and maybe even start brainstorming where Fairlearn might help for those top few scenarios.

I think if tomorrow you all come out with notes on people's impressions of the seed scenarios, and notes on the **Real harms** for three of the scenarios, that'd be great a great step!  Once there's some momentum there, it may become easier to break work up into smaller pieces too so we can make more progress between Thursday meetings. 👍

Reply:
I think that would be fantastic. Let's see how much of this we can do tomorrow :-)